src/Assets/TestProjects/WatchBrowserLaunchApp/WatchBrowserLaunchApp.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>$(CurrentTargetFramework)</TargetFramework>
```

Comment:
Looks like some of the dotnet-watch tests are failing on full Framework.

@pranavkm @mkArtakMSFT, we're working on updating a lot of the tests in this repo so that instead of targeting a fixed version of .NET, they target the current version which will roll forward.  Do you want to look at why these watch tests are failing here, or should we leave them targeting .NET 5?